htdocs/language/english/locale.php

```
@@ -47,17 +47,25 @@
      */
-    public function number_format($number)
+    public function number_format($number, $decimals = null, $decSep = null, $thouSep = null)
     {
-        return number_format($number, 2, '.', ',');
-    }
+        $decimals ??= self::CURRENCY['decimals'];
+        $decSep ??= self::CURRENCY['decSep'];
+        $thouSep ??= self::CURRENCY['thouSep'];
 
+        return number_format((float)$number, $decimals, $decSep, $thouSep);
+    }
```

PHPMD (2.15.0)

[error] The method number_format is not named in camelCase. (undefined)

(CamelCaseMethodName)